source/audioDucking.py

```diff
@@ -171,11 +171,15 @@ def initialize():
 def isAudioDuckingSupported():
 	global _isAudioDuckingSupported
 	if _isAudioDuckingSupported is None:
+		import wx
 		_isAudioDuckingSupported = (
 			config.isInstalledCopy()
 			or config.isAppX
 		) and hasattr(oledll.oleacc, 'AccSetRunningUtilityState')
 		_isAudioDuckingSupported &= systemUtils.hasUiAccess()
+		# If the wx.App has not been initialized, audio ducking callbacks
+		# will fail as they rely on wx.CallLater/wx.CallAfter
+		_isAudioDuckingSupported &= wx.GetApp() is not None

```

Comment:
Perhaps this should happen in / also in `core.callLater`? Here it is a bit removed from the core problem, which might be fine for a quick work-around for this issue / release, but longer term this could happen in other scenarios.

Perhaps as a follow up we should ensure that all usages of `wx.Calllater` are converted to go via `core.callLater

Reply:
What would be the suggested behaviour when we cannot continue in `core.callLater`? Log an exception?

I support those 2 pieces of work for 2022.2

Reply:
Yes, I think log then raise an exception to notify the caller. We'd convert all internal usages of `wx.CallLater` to use `core.callLater` and handle the exception. Then later we can consider an implementation that can work without `wx`.

Reply:
I'd avoid raising an exception: wx raising an exception here is causing the crash.
I think this is because the exception is raised before our exception handling is initialized.
Raising an exception here will not be an improvement of behaviour.

Reply:
I've created a PR here to start on this: https://github.com/nvaccess/nvda/pull/13701

I'd like to get this into rc as a priority first